### DISCUSSION TITLE
Recognition of AQL (ArangoDB Query Language)

### DISCUSSION THREAD

**Discussion by formidablae** [github-linguist/linguist#5487] [Classification] [answered]
Hello,
is there any plan to implement the recognition of files containing AQL ArangoDB query language?
https://www.arangodb.com/docs/devel/aql/fundamentals-syntax.html

**Comment by lildude** [comment, marked as answer]
There's never a plan to implement any new language 😁 - we rely almost purely on community contributions.

If you want it, and it meets our minimum usage requirements, and you have the time and skills, feel free to submit a pull request. See the CONTRIBUTING.md file for details. 

Alternatively, you can open an issue (select "New language") and fill in the details and someone might pick it up and add support at some point in the future.

**Reply by formidablae** [reply]
Thank you, I will check out both options (implement it myself or open a "New language" issue with details) and decide which to do.
